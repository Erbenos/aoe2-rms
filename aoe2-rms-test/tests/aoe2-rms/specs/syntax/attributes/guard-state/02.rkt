#lang aoe2-rms
; Slowly drain a player's food while they do not control the monument.

<PLAYER-SETUP>
(guard-state 'MONUMENT 'AMOUNT_FOOD -5 6)
