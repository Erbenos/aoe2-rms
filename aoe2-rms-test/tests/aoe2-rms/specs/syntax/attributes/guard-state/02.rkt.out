<PLAYER_SETUP>
guard_state MONUMENT AMOUNT_FOOD -5 6
